.github/workflows/multicluster.yaml

```diff
@@ -198,8 +198,16 @@ jobs:
       - name: Clean up GKE
         if: ${{ always() }}
         run: |
-          gcloud container clusters delete ${{ env.clusterName1 }} --zone ${{ env.zone }} --quiet
-          gcloud container clusters delete ${{ env.clusterName2 }} --zone ${{ env.zone }} --quiet
+          i=0
+          for e in ${{ env.clusterName1 }} ${{ env.clusterName2 }}; do
+            gcloud container clusters delete $e --zone ${{ env.zone }} --quiet &
+            pids[$i]=$!
+            ((i++))
+          done
+          # Wait for all cluster-cleanup processes.
+          for pid in ${pids[*]}; do
+            wait $pid
```

Comment:
Why do we even need to wait for the cluster to be deleted?

Reply:
Maybe for error reporting reasons? We'd be able to trace why a cluster was left running

Reply:
It's a background process. If we end the GitHub job, the background process is very likely killed. (Or we just found a way to cheat GitHub workflows.)

Reply:
sure the background process is killed by the API call is already made to google so the cluster will eventually be deleted, no?

Reply:
No, I don't think it works that way. AFAIU, the process makes several calls to the API and thus it may not be finished when it gets killed. If we quit just after the process is started, we also have no guarantee it had time to make even a single API call ;-)